src/components/forms/ProxyDeviceForm.tsx

```diff
@@ -0,0 +1,369 @@
+import { FC } from "react";
+import {
+  Button,
+  Icon,
+  Input,
+  Label,
+  Select,
+  useNotify,
+} from "@canonical/react-components";
+import { useQuery } from "@tanstack/react-query";
+import { queryKeys } from "util/queryKeys";
+import { LxdProxyDevice } from "types/device";
+import { InstanceAndProfileFormikProps } from "./instanceAndProfileFormValues";
+import { fetchProfiles } from "api/profiles";
+import { getInheritedProxies } from "util/configInheritance";
+import Loader from "components/Loader";
+import ScrollableForm from "components/ScrollableForm";
+import RenameDeviceInput from "components/forms/RenameDeviceInput";
+import ConfigurationTable from "components/ConfigurationTable";
+import { MainTableRow } from "@canonical/react-components/dist/components/MainTable/MainTable";
+import { getConfigurationRowBase } from "components/ConfigurationRow";
+import classnames from "classnames";
+import {
+  addNoneDevice,
+  deduplicateName,
+  findNoneDeviceIndex,
+  removeDevice,
+} from "util/formDevices";
+import { getInheritedDeviceRow } from "components/forms/InheritedDeviceRow";
+import { deviceKeyToLabel } from "util/devices";
+import { ensureEditMode } from "util/instanceEdit";
+import NewProxyBtn from "components/forms/NewProxyBtn";
+import ConfigFieldDescription from "pages/settings/ConfigFieldDescription";
+
+interface Props {
+  formik: InstanceAndProfileFormikProps;
+  project: string;
+}
+
+const ProxyDeviceForm: FC<Props> = ({ formik, project }) => {
+  const notify = useNotify();
+
+  const {
+    data: profiles = [],
+    isLoading: isProfileLoading,
+    error: profileError,
+  } = useQuery({
+    queryKey: [queryKeys.profiles],
+    queryFn: () => fetchProfiles(project),
+  });
+
+  if (profileError) {
+    notify.failure("Loading profiles failed", profileError);
+  }
+
+  const inheritedProxies = getInheritedProxies(formik.values, profiles);
+
+  const existingDeviceNames: string[] = [];
+  existingDeviceNames.push(...inheritedProxies.map((item) => item.key));
+  existingDeviceNames.push(...formik.values.devices.map((item) => item.name));
+
+  const addProxy = () => {
+    const copy = [...formik.values.devices];
+    copy.push({
+      type: "proxy",
+      name: deduplicateName("proxy", 1, existingDeviceNames),
```

Comment:
```suggestion
      name: deduplicateName("proxy", 1, existingDeviceNames),
      bind: "host"
```
Either we add a default for the bind field, or we need to add a 3rd option to the dropdown with a "Select value" label, disabled true and an empty value.

Reply:
I added the Select Value option to the dropdown, however the automatically selected option is still Host. I have also added the Select value to the other dropdowns. Please verify whether the behaviour is as you expect, because I would assume we want those Select Value options to show as the defaults and that is not what is currently happeing.

Reply:
What we do in other places is give the "select an option" option a value of empty string, and make the value of the input empty string in case it is undefined.